src/App/Pages/Settings/SecuritySettingsPageViewModel.cs

```diff
@@ -305,6 +328,7 @@ private async Task InitVaultTimeoutPickerAsync()
             {
                 _customVaultTimeoutTime = TimeSpan.FromMinutes(vaultTimeout);
             }
+            MainThread.BeginInvokeOnMainThread(() => { TriggerPropertyChanged(nameof(IsVaultTimeoutActionLockAllowed)); });
```

Comment:
⛏️ No need for brackets
```suggestion
            MainThread.BeginInvokeOnMainThread(() => TriggerPropertyChanged(nameof(IsVaultTimeoutActionLockAllowed)) );
```